app/Config.hs

```diff
@@ -113,7 +116,12 @@ logMAccessShort host user _ requestLine responseCode _ referer _ = do
 
 -- | The path to the database file, relative to the project root.
 databasePath :: IO Text
-databasePath = databasePathValue <$> loadConfig
+databasePath = do
```

Comment:
The configuration change shouldn't be here, it should be in `loadConfig` itself. You should be creating a *second yaml file*, not adding a new key to the original yaml file. This achieves the behaviour of overriding the default database path with a new one used for testing purposes.